ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/carousel/clientlibs/less/carousel.less

```diff
@@ -13,7 +13,7 @@
         .image {
           img {
             max-width: 100%;
-            max-height: 300px;
+            max-height: 500px;
```

Comment:
@uditnaagar777 : apart from max-height do we not need height:auto as well ?

Reply:
@rishabh-behl The images in the carousel have different aspect ratio. So after adding height:auto, the carousel height is changing on the page when the next image slides in.So i have omitted it.